tests/common_testing/alfasim_sdk_common_testing/filled_case_descriptions.py

```diff
@@ -424,7 +424,9 @@
     pvt_model="gavea",
     top_node="mass_source_node",
     initial_conditions=INITIAL_CONDITIONS_DESCRIPTION,
-    gas_lift_valve_equipment={"My gas-lift valve": GAS_LIST_VALVE_DESCRIPTION},
+    equipment=case_description.AnnulusEquipmentDescription(
```

Comment:
I think it is better to use the existing description here.
```suggestion
    equipment=ANNULUS_EQUIPMENT_DESCRIPTION,
```